.codecov.yml

```diff
@@ -10,8 +10,12 @@ comment:
 coverage:
   status:
     project:
-      # Don't fail the status check if coverage decreases.
-      informational: true
+      default:
+        # Don't fail the status check if coverage decreases.
+        informational: true
     patch:
-      # Don't fail the status check if coverage decreases.
-      informational: true
+      default:
+        # Don't fail the status check if coverage decreases.
+        informational: true
+github_checks:
+  annotations: false
```

Comment:
This disables the following annotations, which IMO are just noise:

<img width="1096" alt="Screenshot 2024-05-21 at 09 36 07" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/5a412718-345f-46cb-b3db-bc663ed8ce4c">